Resources/Prototypes/_NF/Catalog/Fills/Items/briefcases.yml

```
@@ -60,7 +60,7 @@
 - type: entity
   parent: SpaceCashExpedition
   id: SpaceCashHazardousExpedition
-  name: spesos (40000)
+  name: spesos (40.000)
```

Consistency with en-US cultureinfo.
```suggestion
  name: spesos (40,000)
```